server/db/schema.js

```diff
@@ -73,6 +76,33 @@ const schema = {
       )`,
     },
   ],
+  psiRelationalTables: [
```

Comment:
These should be able to be entered in the same array as the previous tables shouldn't it? What was the issue again?

Reply:
relational tables are processed first, so they can't refer to a document table that doesn't exist yet. Bypassing this throws errors every time we spin up a new instance of the project and rerun old migrations (every time we run the test suite).

Reply:
Ahhh, thanks for reiterating that for me, I understand now